Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Input/Controllers/OculusQuestHand.cs

```diff
@@ -337,10 +341,30 @@ protected bool UpdateHandData(OVRHand ovrHand, OVRSkeleton ovrSkeleton)
                 }
             }
 
+            // Pinch was also used as grab, we want to allow hand-curl grab not just pinch.
+            // Determine pinch and grab separately
+            MixedRealityPose wristPose, indexKnucklePose, indexTipPose;
+            if (jointPoses.TryGetValue(TrackedHandJoint.Wrist, out wristPose))
```

Comment:
Can you extract this out into a function? Would make this more readable.

Reply:
sure - where would you like it?  really it should go into something like a HandUtils script, but I don't think there is one in MRTK-Quest

Reply:
It can be in the same class for now, just to have it as a function would help refactor later.